api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxPvlsCohortQueries.java

```diff
@@ -133,10 +135,11 @@ public CohortDefinition getPatientsWithViralLoadResults() {
 		cd.addParameter(new Parameter("location", "Location", Location.class));
 		String mappings = "startDate=${startDate},endDate=${endDate},location=${location}";
 		cd.addSearch("results", EptsReportUtils.map(hivCohortQueries.getPatientsViralLoadWithin12Months(), mappings));
-		cd.addSearch("baseCohort", EptsReportUtils.map(genericCohortQueries.getBaseCohort(), mappings));
+		// cd.addSearch("baseCohort",
```

Comment:
Rather remove these instead of commenting these lines out if they are not being used.